templates/svelte/entry-type/ui/src/{{dna_role_name}}/{{coordinator_zome_manifest.name}}/{{#each entry_type.fields}}{{#if (and linked_from (not (eq linked_from.hash_type 'AgentPubKey') ) )}}{{pascal_case (plural ..¡entry_type.name)}}For{{pascal_case linked_from.name}}.svelte{{¡if}}{{¡each}}.hbs

```diff
@@ -3,38 +3,53 @@ import { onMount, getContext } from 'svelte';
 import '@material/mwc-circular-progress';
 import type { Link, ActionHash, EntryHash, AppAgentClient, Record, AgentPubKey, NewEntryAction } from '@holochain/client';
 import { clientContext } from '../../contexts';
-import type { {{pascal_case ../entry_type.name}} } from './types';
+import type { {{pascal_case ../entry_type.name}}, {{pascal_case ../coordinator_zome_manifest.name}}Signal } from './types';
 import {{pascal_case ../entry_type.name}}Detail from './{{pascal_case ../entry_type.name}}Detail.svelte';
 
 export let {{camel_case linked_from.singular_arg}}: {{linked_from.hash_type}};
 
 let client: AppAgentClient = (getContext(clientContext) as any).getClient();
 
-let links: Array<Link> | undefined;
+let hashes: Array<ActionHash> | undefined = [];
 
-let loading = true;
+let loading: boolean;
 let error: any = undefined;
 
-$: links, loading, error;
+$: hashes, loading, error;
 
 onMount(async () => {
   if ({{camel_case linked_from.singular_arg}} === undefined) {
     throw new Error(`The {{camel_case linked_from.singular_arg}} input is required for the {{pascal_case (plural ../entry_type.name)}}For{{pascal_case linked_from.name}} element`);
   }
 
+  await fetch{{pascal_case (plural ../entry_type.name)}}();
+
+  client.on('signal', signal => {
+    if (signal.zome_name !== '{{../coordinator_zome_manifest.name}}') return;
+    const payload = signal.payload as {{pascal_case ../coordinator_zome_manifest.name}}Signal;
+    if (payload.type !== 'EntryCreated') return;
+    if (payload.app_entry.type !== '{{pascal_case ../entry_type.name}}') return;
+    if (JSON.stringify(payload.app_entry.{{snake_case linked_from.singular_arg}}) !== JSON.stringify({{camel_case linked_from.singular_arg}})) return;
```

Comment:
I use `isEqual` from lodash https://lodash.com/docs/4.17.15#isEqual - would be nice to avoid the external dep though

Reply:
I thought about using this too, but I wanted to avoid adding more dependencies. Still, I'm thinking I could just install `lodash.isEqual` by itself, and it's not too hefty anyway, right?

Reply:
@mattyg I have opted for a simpler solution of refetching on `signal`